packages/backend/src/routes/dons.ts

```diff
@@ -0,0 +1,77 @@
+import express, { Request } from 'express';
+import prisma from '@/lib/prismaClient.js';
+
+interface Topping {
+  id: number,
+  amount: number,
+}
+
+interface PriceBody {
+  size: number,
+  toppings: Topping[] | undefined,
+}
+
+const router = express.Router();
+
+router.post('/price', async (req: Request<any, any, PriceBody>, res, next) => {
+  const size = req.body.size;
+  const toppings = req.body.toppings || [];
```

Comment:
```suggestion
  const reqSize = req.body.size;
  const reqToppings = req.body.toppings || [];
```
[domino] `size` 変数を prisma の戻り値に使いたい関係上